### PR TITLE
 [MIG] account_credit_control_dunning_fees: Migration to 12.0 

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: account_credit_control_dunning_fees/model/run.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+class CreditControlRun(models.Model):
+    """Add computation of fees"""
+
+    _inherit = "credit.control.run"
+
+    @api.multi
+    @api.returns('credit.control.line')
+    def _generate_credit_lines(self):
+        """Override method to add fees computation"""
+        credit_lines = super(CreditControlRun, self)._generate_credit_lines()
```
Comment (i-vyshnevska) [inline account_credit_control_dunning_fees/model/run.py]:
```suggestion
        credit_lines = super()._generate_credit_lines()
```

File: account_credit_control_dunning_fees/tests/test_fees_generation.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# Copyright 2014-2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests import common
+
+
+class FixedFeesTester(common.TransactionCase):
+
+    def setUp(self):
+        """Initialize credit control level mock to test fees computations"""
+        super(FixedFeesTester, self).setUp()
```
Comment (i-vyshnevska) [inline account_credit_control_dunning_fees/tests/test_fees_generation.py]:
```suggestion
        super().setUp()
```

File: account_credit_control_dunning_fees/tests/test_fees_generation.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        self.currency_model = self.env['res.currency']
+        self.euro = self.currency_model.search([('name', '=', 'EUR')])
+        self.assertTrue(self.euro)
+        self.usd = self.currency_model.search([('name', '=', 'USD')])
+        self.assertTrue(self.usd)
```
Comment (i-vyshnevska) [inline account_credit_control_dunning_fees/tests/test_fees_generation.py:15]: line 11-15 are weird

File: account_credit_control_dunning_fees/tests/test_fees_generation.py [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+from odoo.tests import common
+
+
+class FixedFeesTester(common.TransactionCase):
+
+    def setUp(self):
+        """Initialize credit control level mock to test fees computations"""
+        super(FixedFeesTester, self).setUp()
+        self.currency_model = self.env['res.currency']
+        self.euro = self.currency_model.search([('name', '=', 'EUR')])
```
Comment (i-vyshnevska) [inline account_credit_control_dunning_fees/tests/test_fees_generation.py:12]: why do not use self.env.ref('base.EUR')

File: account_credit_control_dunning_fees/model/line.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+class CreditControlLine(models.Model):
+    """Add dunning_fees_amount_fees field"""
+
+    _inherit = "credit.control.line"
+
+    dunning_fees_amount = fields.Float(string='Fees')
+    balance_due_total = fields.Float(string='Balance due with fees',
+                                     compute='compute_balance_due')
+
+    @api.one
```
Comment (i-vyshnevska) [inline account_credit_control_dunning_fees/model/line.py]: deprecated

File: account_credit_control_dunning_fees/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
+{'name': 'Credit control dunning fees',
+ 'version': '12.0.1.0.0',
+ 'author': "Camptocamp, Odoo Community Association (OCA)",
+ 'maintainer': 'Camptocamp',
+ 'category': 'Accounting',
+ 'complexity': 'normal',
+ 'depends': ['account_credit_control'],
+ 'website': 'http://www.camptocamp.com',
```
Comment (i-vyshnevska) [inline account_credit_control_dunning_fees/__manifest__.py]:
```suggestion
 'website': 'https://odoo-community.org/',
```

**Review by i-vyshnevska — APPROVED** [review]
base work LG, just nit-picking

**Comment by zeroheure** [comment]
Any news on this ?

**Pull request closed without merging**